README.md

```diff
@@ -84,7 +84,7 @@ To run this sample, you'll need:
 ## Steps to Run the app
 
 > **Note:**
-> This sample ships with a default `redirect_uri` configured in the `AndroidManifest.xml`. In order for the default `redirect_uri` to work, this project must be built with the `debug.keystore` located in the `gradle/` directory. To configure signing in Android Studio, see [Sign Your App](https://developer.android.com/studio/publish/app-signing).
+> This sample ships with a default `redirect_uri` configured in the `AndroidManifest.xml`. In order for the default `redirect_uri` to work, this project must be built with the `debug.keystore` located in the `gradle/` directory. See [Sign your debug build](https://developer.android.com/studio/publish/app-signing#debug-mode) for more info.
 

```

Comment:
Will they need to follow the same steps I'm doing in the quickstart (copying over the debug.keystore from the sample over their home/ keystore?
What are you having them do by doing to [sign your debug build]?

Reply:
I think they will still have to do as you said, which is to copy the debug keystore to the default location as you stated. The link to the doc kinda refers to that but doesn't actually clearly explain what needs to happen to be able to achieve that. I think may be we can move away from this, and instead of making the users perform this copy over of the keystore to the home location, we can instead hard code the location of the keystore in the `build.gradle` file. I believe that is what @shoatman did to the [kotlin sample](https://github.com/Azure-Samples/ms-identity-android-kotlin/blob/7e4cdd59aac02930503695fdff4646a82f1c2190/app/build.gradle#L24-L31)

@rpdome Is this something we can do with the Android sample too?

Reply:
I like that idea! It'd make the quickstart a breeze!  Plus, it seems like bad karma to have a quickstart instruct the user to modify their default keystore...